splash-dao-offchain/src/routines/inflation/mod.rs

```diff
@@ -109,14 +113,18 @@ pub struct Behaviour<
     funding_box: FB,
     /// Backlog of unspent `make_voting_escrow_order` UTxOs
     mve_order_backlog: MVE,
+    /// Backlog of unspent `extend_voting_escrow_order` UTxOs
+    eve_order_backlog: EVE,
```

Comment:
Why not a single backlog for a sum of all orders?

Reply:
Right, it's done